621cc3b0d2fe11f6ac7d6f980c5aba9591d171bf
sam.noedel@gmail.com
Sam Noedel
1549754966

diff --git a/src/index.ts b/src/index.ts
index f363817..0026f68 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -22,6 +22,8 @@ let args: Args = yargs

 // stream which emits commit hashes
 commitStream(args)
-  .subscribe((data) => {
-    console.log('data', data)
-  })
+  .subscribe(
+    data => console.log('data', data),
+    err => console.error(err),
+    () => console.log('done')
+  )
